### PR TITLE
Added quotes to "state" field value.

### PR DESCRIPTION
**Pull request by akirsanovkg**

### CODE REVIEW

File: core/block.go [hunk 1]
```diff
@@ -533,7 +533,7 @@ func (block *Block) Seal() error {
 }
 
 func (block *Block) String() string {
-	return fmt.Sprintf("{\"height\":%d, \"hash\":\"%s\", \"parentHash\":\"%s\", \"state\": %s, \"nonce\":%d, \"timestamp\": %d, \"coinbase\": \"%s\", \"tx\": %d}",
+	return fmt.Sprintf("{\"height\":%d, \"hash\":\"%s\", \"parentHash\":\"%s\", \"state\": \"%s\", \"nonce\":%d, \"timestamp\": %d, \"coinbase\": \"%s\", \"tx\": %d}",
```
Comment (yeouchien) [inline core/block.go]: https://golang.org/pkg/fmt/
`%q	a double-quoted string safely escaped with Go syntax`
`\"%s\"` can be replaced by just `%q`

**Comment by fbzhong** [comment]
Nice commit, thank you guys!

**Comment by qywang2012** [comment]
ooh, the string function should return a JSON-format string, so I add quotation marks.

**Comment by fbzhong** [comment]
Lol, maybe it’s better to define a new func named JSON() for your requirements.

**Pull request merged**